docs/api.md

```diff
@@ -0,0 +1 @@
+See [api/README.md](api/README.md).
```

Comment:
The hyperlink doesn't work. Perhaps try `/api/README.md`?

Reply:
Ha, I had the slash, but didn't like how it looked, and removed both :)